### PR TITLE
pipewire: apply qjackctl fix from master

### PR DESCRIPTION
**Pull request by st3r4g**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/696

### CODE REVIEW

File: srcpkgs/pipewire/patches/3450bea4167573d3d364e1a768605d11be241293.diff [hunk 1]
```diff
@@ -0,0 +1,73 @@
+diff --git a/pipewire-jack/src/pipewire-jack.c b/pipewire-jack/src/pipewire-jack.c
+index 5163d4a5e287d076160ee01bb8c1c1aac73ff35c..a7ead586db3d682b7ae0e6ba0069fec98458d69b 100644
+--- a/pipewire-jack/src/pipewire-jack.c
++++ b/pipewire-jack/src/pipewire-jack.c
```
Comment (ericonr) [inline srcpkgs/pipewire/patches/3450bea4167573d3d364e1a768605d11be241293.diff:4]:
```suggestion
--- pipewire-jack/src/pipewire-jack.c
+++ pipewire-jack/src/pipewire-jack.c
```

Avoids the need for the series file.

**Review by ericonr — COMMENTED** [review]
Can you add a link to the commit to the top of the file. It's hard to tell what this does.

**Comment by st3r4g** [comment]
> Can you add a link to the commit to the top of the file. It's hard to tell what this does.

Well, the commit hash is already in the filename, I though it was descriptive enough. Also it will be removed on next release.

**Comment by ericonr** [comment]
> Well, the commit hash is already in the filename, I though it was descriptive enough. 

I somehow missed that, but the link is also easier to click and go directly to the explanation.

Re. Waiting for release, your choice. Pipewire is small enough IMO that either is ok. Unless a release is coming in the next few days, then I guess waiting is best.

**Comment by st3r4g** [comment]
Rerun CI because GitHub had problem earlier today.

> Unless a release is coming in the next few days, then I guess waiting is best.

Releases are usually every 2 weeks.

**Pull request merged**